cmd/client.go

```diff
@@ -174,13 +172,3 @@ func createClient() (*client.Client, error) {
 
 	return client.New(user, pass, "localhost", port)
 }
-
-func printJsonResults(results interface{}) error {
```

Comment:
This JSON formatting functionality seems to be repeated throughout the PR. Any reason in particular to not have it live in this helper function?

Reply:
Nice catch, I pulled it out into the util package.